scripts/post.js

```diff
@@ -343,7 +343,14 @@ async function addInterLinks() {
     let bStart = Date.now();
     const keywords = (Array.isArray(json) ? json : json.data)
       // scan article to filter keywords down to relevant ones
-      .filter(({ Keyword }) => document.querySelector('main').textContent.toLowerCase().indexOf(Keyword) !== -1);
+      .filter(({ Keyword }) => document.querySelector('main').textContent.toLowerCase().indexOf(Keyword) !== -1)
+      // prepare regexps
+      .map((item) => {
+        return {
+          regexp: new RegExp(`\\b(${item.Keyword})\\b`, 'iu'),
```

Comment:
just hope, they don't put `.*` in the keyword :-)

Reply:
see https://stackoverflow.com/questions/3561493/is-there-a-regexp-escape-function-in-javascript